mod_form.php

```diff
@@ -56,6 +56,28 @@ public function definition() {
             $nexturl = $PAGE->url;
             zoom_fatal_error($errstring, 'mod_zoom', $nexturl, $config->zoomurl);
         }
+        
+        // Get list of schedule for users if supported.
+        // List of users who can use Zoom mod in this class.
+        $moodleusers = get_enrolled_users(context $context, 'mod/zoom:addinstance', 0, $'u.*', 'lastnane');
```

Comment:
I'm not sure on this one, but why are we specifying the variable type if the function definition is responsible for that? Also, maybe a stray `$` and field typo?
```suggestion
        $moodleusers = get_enrolled_users($context, 'mod/zoom:addinstance', 0, 'u.*', 'lastname');
```

Reply:
Sorry rogue, since I copied the function call's signature from somewhere else.